### PR TITLE
[RELEASE ⛰️ ] v1.10.0-RC.0

### PR DESCRIPTION
**Pull request by W3stside**
# DO NOT SQUASH

## DESCRIPTION

Claim stuff

This time hidden on: prod, barn and ens. Still visible everywhere else, including staging.

## CHANGELOG

6eee223bf Removed unused flag IS_CLAIMING_ENABLED (it's defined elsewhere)
d1fbb50b1 Added full amounts on hover for all claims values (#2282)
3bcd42f9e Remove cow claim button desktop. (#2292)
435874c27 Disable claiming: BARN / PROD / ENS (#2298)
22eb95579 Disable vCow balance dropdown on profile page (#2291)
7355deb5b [Claim] Revoke Approval hook (#2249)
a8d90c6aa [Claim-Approve] Hook optional amountToCheckAgainstAllowance param (#2175)
19adb51a5 Check claims disconnected (#2281)
9b3cde3ec Pr2256/follow up (#2279)
f58f3f081 vCOW EVERYWHERE! (not vCoW) (#2283)
49425bf95 Set initial claim invest input amount to an empty string (#2280)
1a0e727ab Bump nanoid from 3.1.30 to 3.2.0 (#2257)
456c121c4 Responsive balance indicator. (#2272)
f4dd3476c Add link read more in footer, and inform about changing networks (#2265)
142baaf3a add paragraphs (#2278)
88c3ce008 Add token to metamask (#2267)
202db8050 Change text (#2268)
bdb4403d4 Add link to the profile (#2269)
71ecc4499 Simplify links (#2266)
b7f82f0f6 Fix Gnosis Safe claiming Workflow (#2277)
35ba6b36f Add badges and link to etherscan (#2264)
aa5a8a273 Add moo sounds (#2259)
b1335dc81 Warning claim too much eth (#2256)
f4ae1602c Claimed amount on success screen (#2253)
f5fe2486b Dropdown component for vCOW balance  (#2156)
2ffd5fee8 Add Recipient toggle (#2013)
007320048 Merge branch 'master' into develop
bb7b1a15b Lowercasing searched address to find it on chunk file (#2260)
ca3b4bd41 Claim invest general error msg (#2254)
b9523afa4 Deployment in all networks (#2258)
4603884cd Disable claim review on zero or not approved (#2250)
6499a5b1c Add external links to const. (#2252)
b3c8e1be9 Merge branch 'master' into develop
56fbeb10f Create and implement a Stepper component. (#2246)
26e7540f9 Disable claim review on errors (#2241)
5bbc4d87c Fix duplicate CSS properties. (#2243)
a76e66860 Consistent locale on claim (#2240)
7b5ed4fc2 Total vcow on investment summary (#2237)
77a9d5762 Change account fixes (#2238)
d0cfce851 Pr2235/follow up (#2239)
edac11d3b Fix for investment amount reset when you go back (#2247)
0132fe7f6 Update address error msg text (#2248)
0a41de342 Claim partial investments (#2234)
b4ea09547 Pr2232/follow up (#2236)
d775d9497 Claim pending icon spinner and show timezone. (#2235)
81be3614a Fixes the process console error (#2231)
ef171d238 Claim style improve 20 (#2232)
9f4605d91 Merge branch 'claim' into develop
ffb70c867 Hide claim button for claiming/claimed states (#2222)
9367f949f Fix for selectedAll field (#2226)
9615e5f8c Add link claiming (#2224)
eca3a889b Simplify error messages (#2217)
172b7ce22 Claim instead of claim (#2220)
5aab63427 Misc styling (#2228)
a116ab5dd Claim last investment page (#2227)
e8dbe0931 Change contract (#2210)
f35c12f55 [CLAIM] Button refactor and error fix (#2218)
5d43dc5ba Merge branch claim-investment-flow-3 into claim
8c95bbade Fix error on bad merge
2146760b5 Improve input field claim (#2186)
391f26c12 Revert "New claim contract abi (#2194)"
a657285d0 Created type for price fn names (#2205)
77518659c [Claim-Reject-Modals] Fixes empty modal (#2204)
7099b6f11 [Claim] Rejected/failed/errors in claim flow (#2169)
7029923a1 Claim change account (#2173)
a57a28381 New claim contract abi (#2194)
842d37d8f Claim investment (#2191)
1c15ca524 Claim dynamic prices (#2182)
45c0b23ca Merge pull request #2163 from gnosis/claim-investment-flow-2
95fa842cb Invest amount calculation update
fd056b1ca Merge branch 'claim' into claim-investment-flow-2
9f448bfd2 Merge branch 'claim-investment-flow-2' of github.com:gnosis/cowswap into claim-investment-flow-2
704c508f2 Update for initInvestFlowData
172fe4321 Rename constacts (#2180)
8502d80e6 Claim style improve 10 (#2174)
ba6ff22f1 Fix bad merge (#2184)
6277b0340 Use airdrop claim end date (#2178)
1599ccf92 2120/refactor time hooks (#2179)
b7f6eeed1 Pr2160/follow up (#2176)
d90b526da Merge branch 'claim' into claim-investment-flow-2
46410ccc6 PR updates
325efc170 2120/claim countdowns (#2158)
706011ec2 remove progress bar from Profile page (#2177)
0b6b4b7d0 Fix native investment (#2123)
2f2f1cb94 Wording change and CSS cleanup. (#2170)
00195a958 new progressbar component of claim branch && more values available (#2060)
edd210a2f Merge branch 'master' into develop
af68256bf [Claim - ClaimTable - Pending] - Show loader and minor style change (#2162)
1346e0ba7 Updated invest percentage logic
345c482c0 Moved invest data redux
a07c6965a WIP styling investment flow. (#2152)
4c41c5791 2139/claim identicon (#2161)
d1cd9a6af Fix free claims (#2160)
2830c0000 Fixing assumtion regarding data format stored on redux (#2159)
5d21047f8 [Claim Approve] Approve button async logic and styles (#2155)
37fcffdb2 Use theme colors. (#2113)
e492c2196 Small claim fixes (#2153)
bd559e6a5 [Claim - Approve] -> USDC (#2154)
84a384d6e [Claim refactor] - Split InvestmentFlow and minor slider impl. (#2144)
9899670f4 Replace Gnosis Protocol --> CoW Protocol (#2140)
75d4d3366 Change redux store on success claiming (#2141)
f560da018 Add claim finalized middleware (#2126)
8ef567f5a [Claim refactor] Table / InvestmentFlow / Status (#2138)
f3d027b37 get the chainId specific GNO (#2143)
2deafd347 [Claim] Approve flow - GNO approve (#2130)
51cde4fa3 Refactor claim timestamp hooks (#2127)
f2765ea0a Updating vCOW contract address to 0x71A377EC0026A8E35AfD082d54E2f62d39AF075c (#2135)
8a83dd9d7 [CLAIM] Pending claim UI (#2049)
ddf8ca607 Update xdai and wxdai token logo (#2122)
fee0156bd Merge pull request #2124 from gnosis/merge-develop-to-claim
5d7aa6193 Merge branch 'develop' into claim
be65cac4e vCOW on profile (#2062)
ab29d42b8 Merge pull request #2086 from gnosis/update-contract
9af6cbf53 Update contract
224dab305 Claim refactor index redux (#2077)
f92674baf [Claim refactor index] - Part 1 - Extrapolate 5 small components (summary, nav, claim messages) (#2072)
48d1b7abe [Claim] Context state (#2064)
efc1af09a Claim hooks 4 exclude expired claims (#2059)
6272c54e0 [Claim] Profile: vCOW balance (#2051)
0bd385826 Pr2052/follow up (#2055)
efcb9ad96 Claim hooks 3 (#2056)
309753c83 Updated claim enum and fix language selector
42e307aa2 Updated claim enum usage and some other small changes
319d7a940 Claim page 2 rebased (#2054)
c053cf20f Real airdrop repo (#2052)
8d53e6bad Claim hooks 2 - ready to fetch (#2043)
956c342ea Claim flow styles (#2042)
d2060bb1b Claim hooks pt 1 - useClaimCallback (#2041)
52b416fd4 Claim hooks part 0 (#2032)
b1628bbd8 vCow contracts (#2029)
fccf88468 Added ClaimType type (#2025)
325e3eac7 Basic claim (#2000)

### CODE REVIEW

**Review by anxolin — COMMENTED** [review]
~If this is just the 2 PRs we reviewed, u can just merge to master, and create a new RC~ 
*david edit: probably not for a little

File: src/components/claim/ClaimModal.tsx [hunk 1]
```diff
@@ -51,23 +51,25 @@ export default function ClaimModal() {
   const isOpen = useModalOpen(ApplicationModal.SELF_CLAIM)
   const toggleClaimModal = useToggleSelfClaimModal()
 
-  const { account, chainId } = useActiveWeb3React()
+  const { chainId } = useActiveWeb3React()
 
+  const account = '0x0010B775429d6C92333E363CBd6BF28dDF1A87E6'
```
Comment (alfetopito) [inline src/components/claim/ClaimModal.tsx:56]: @alfetopito revert changes

File: src/components/swap/styleds.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { loadingOpacityMixin } from 'components/Loader/styled'
```
Comment (alfetopito) [inline src/components/swap/styleds.tsx:1]: Likely modified the wrong file.
Should have created a mod but instead changes were applied to original files

File: src/custom/components/ProgressBar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import { ProgressBarWrap, ProgressContainer, Progress, Label, FlexWrap, HiddenRange, ProgressVal } from './styled'
```
Comment (alfetopito) [inline src/custom/components/ProgressBar/index.tsx:1]: Unfortunately is not in use, remove

File: src/custom/components/ProgressBar/styled.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components/macro'
```
Comment (alfetopito) [inline src/custom/components/ProgressBar/styled.tsx:1]: Unfortunately is not in use, remove

File: src/custom/constants/index.ts [hunk 5]
```diff
@@ -10,6 +10,7 @@ export const INITIAL_ALLOWED_SLIPPAGE_PERCENT = new Percent('5', '1000') // 0.5%
 export const RADIX_DECIMAL = 10
 export const RADIX_HEX = 16
 
+// TODO: remove, this is duplicated with `import { ONE_HUNDRED_PERCENT } from 'constants/misc'`
```
Comment (alfetopito) [inline src/custom/constants/index.ts:13]: Remove duplicated constant

File: src/custom/state/claim/hooks/mocks/claimData.ts [hunk 6]
```diff
@@ -0,0 +1,153 @@
+import { RepoClaimData } from '..'
```
Comment (alfetopito) [inline src/custom/state/claim/hooks/mocks/claimData.ts:1]: Remove mock data

**Review by alfetopito — APPROVED** [review]
@gnosis/gp-frontend left comments regarding files/changes that should be reverted/changed/updated

Merging now

**Pull request merged**